lib/ansible/modules/monitoring/zabbix_host.py

```diff
@@ -239,6 +247,9 @@ def update_host(self, host_name, group_ids, status, host_id, interfaces, exist_i
                 parameters['proxy_hostid'] = proxy_id
             if visible_name:
                 parameters['name'] = visible_name
+            if description:
```

Comment:
i have tested with ```if description is not None:``` but it is not the desired result.

I think we should leave it as it is. so the behavior is the same for all options.